docs/book/src/user/quick-start.md

```diff
@@ -228,6 +228,11 @@ clusterctl init --infrastructure openstack
 
 Please visit the [Metal3 project][Metal3 getting started guide].
 
+{{#/tab }}
+{{#tab Packet}}
```

Comment:
Packet should be added to the list of tabs at line 130

Reply:
Can I change it now? Because you approved it, not sure if that will drive the automation crazy or not